File: src/main/java/net/atomique/ksar/xml/StatConfig.java
package net.atomique.ksar.xml;

public class StatConfig {

    public StatConfig(String s) {
        StatName = s;
    }

    public String getGraphName() {
        return GraphName;
    }

    public void setGraphName(String GraphName) {
        this.GraphName = GraphName;
    }

    public String getHeaderStr() {
        return HeaderStr;
    }

    public void setHeaderStr(String s) {
        HeaderStr = s;
    }

    public String getStatName() {
        return StatName;
    }

    public boolean canDuplicateTime() {
        return duplicatetime;
    }

    public void setDuplicateTime(String s) {
        if ( "yes".equals(s) || "true".equals(s) ){
            duplicatetime=true;
        }
    }


    private String StatName = null;
    private String GraphName = null;
    private String HeaderStr = null;
    private boolean duplicatetime = false;
}

File: src/main/kotlin/net/atomique/ksar/xml/StatConfig.kt
/*
 * Copyright 2018 The kSAR Project. All rights reserved.
 * See the LICENSE file in the project root for more information.
 */
package net.atomique.ksar.xml

class StatConfig(var statName: String) {
    var graphName: String? = null
    var headerStr: String? = null
    private var duplicatetime = false

    fun canDuplicateTime(): Boolean = duplicatetime

    fun setDuplicateTime(s: String) {
        if ("yes" == s || "true" == s) {
            duplicatetime = true
        }
    }
}
